var/spack/repos/builtin/packages/asagi/package.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+#
+# Author: Gilbert Brietzke
+# Date: July 2, 2019
+
+from spack import *
+
+
+class Asagi(CMakePackage):
+    """a pArallel Server for Adaptive GeoInformation."""
+
+    homepage = "https://github.com/TUM-I5/ASAGI"
+    git = "https://github.com/TUM-I5/ASAGI.git"
+
+    version('1.0.1', commit='f633f96931ae00805f599078d5a1a6a830881554',
+            submodules=True, preferred=True)
+    version('1.0', commit='f67250798b435c308b9a1e7516f916f7855534ec',
+            submodules=True)
```

Comment:
v1.0 should be added using the URL:
   https://github.com/TUM-I5/ASAGI/archive/v1.0.tar.gz

No need to do git download for that.

Reply:
but I think a submodule is missing within the tar.
Using git with submodules=True resolves this without extra efforts

Reply:
OK thanks.  Please add a comment above `version('1.0'...)` explaining this.

Reply:
added comments